src/graph/UpdateEdgeExecutor.cpp

```diff
@@ -105,7 +105,7 @@ Status UpdateEdgeExecutor::prepareSet() {
         updateItems_.emplace_back(std::move(updateItem));
     }
     if (updateItems_.empty()) {
-        status = Status::Error("There must be some correct update items.");
+        status = Status::SyntaxError("There must be some correct update items.");
     }
```

Comment:
Why should we care? I don't think that will happen. WDYT?

Reply:
This problem has been considered in the parser.yy file, correct? 
```

update_list
    : update_item {
        $$ = new UpdateList();
        $$->addItem($1);
    }
    | update_list COMMA update_item {
        $$ = $1;
        $$->addItem($3);
    }
    ;

update_item
    : name_label ASSIGN expression {
        $$ = new UpdateItem($1, $3);
    }
    | alias_ref_expression ASSIGN expression {
        $$ = new UpdateItem($1, $3);
        delete $1;
    }
    ;
```

Reply:
Good catch, thx.